* fixes file2alias cross compiling, as it compiles at host side.
(there is no asm/types.h)

diff -Nur linux-2.6.18.orig/include/linux/input.h linux-2.6.18/include/linux/input.h
--- linux-2.6.18.orig/include/linux/input.h	2006-09-17 16:18:59.000000000 +0200
+++ linux-2.6.18/include/linux/input.h	2006-09-17 16:19:58.000000000 +0200
@@ -16,7 +16,6 @@
 #include <sys/time.h>
 #include <sys/ioctl.h>
 #include <sys/types.h>
-#include <asm/types.h>
 #endif
 
 /*
diff -Nur linux-2.6.18.orig/scripts/mod/file2alias.c linux-2.6.18/scripts/mod/file2alias.c
--- linux-2.6.18.orig/scripts/mod/file2alias.c	2006-09-17 16:19:52.000000000 +0200
+++ linux-2.6.18/scripts/mod/file2alias.c	2006-09-17 16:20:39.000000000 +0200
@@ -29,6 +29,9 @@
 
 #include <ctype.h>
 
+typedef int32_t	__s32;
+typedef int16_t	__s16;
+typedef char		__s8;
 typedef uint32_t	__u32;
 typedef uint16_t	__u16;
 typedef unsigned char	__u8;
